src/profile.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class MyForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      age: null,
+      bio: '',
+      gender: 'Female'
+    };
+  }
+
+
+  myChangeHandler = (event) => {
+    console.log(event.target.files[0])
+    let nameField = event.target.name;
+    let fieldVal = event.target.fieldValue;
+    let errMsg = '';
+    if (nameField === "age") {
+      if (fieldVal != "" && !Number(fieldVal)) {
+        errMsg = <strong>infieldValid input, please enter a number</strong>;
+      }
+    }
+    this.setState({errormessage: errMsg});
+    this.setState({[nameField]: fieldVal});
+  }
+
+  handleUpdate = (event)=> {
+      event.preventDefault();
+      alert("Your profile has been updated!")
```

Comment:
this is where you should put the API call later to attach it to python

Reply:
thanks for the tip